packages/shared/forks/ReactFeatureFlags.test-renderer.native.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import typeof * as FeatureFlagsType from 'shared/ReactFeatureFlags';
+import typeof * as ExportsType from './ReactFeatureFlags.test-renderer';
+
+export const debugRenderPhaseSideEffectsForStrictMode = false;
+export const enableDebugTracing = false;
+export const enableSchedulingProfiler = false;
+export const enableSchedulingProfilerComponentStacks = false;
+export const warnAboutDeprecatedLifecycles = true;
+export const replayFailedUnitOfWorkWithInvokeGuardedCallback = false;
+export const enableProfilerTimer = __PROFILE__;
+export const enableProfilerCommitHooks = false;
+export const enableSchedulerTracing = __PROFILE__;
+export const enableSuspenseServerRenderer = false;
+export const enableSelectiveHydration = false;
+export const enableBlocksAPI = false;
+export const enableLazyElements = false;
+export const disableJavaScriptURLs = false;
+export const disableInputAttributeSyncing = false;
+export const enableSchedulerDebugging = false;
+export const enableDeprecatedFlareAPI = false;
+export const enableFundamentalAPI = false;
+export const enableScopeAPI = false;
+export const enableCreateEventHandleAPI = false;
+export const warnAboutUnmockedScheduler = false;
+export const enableSuspenseCallback = false;
+export const warnAboutDefaultPropsOnFunctionComponents = false;
+export const warnAboutStringRefs = false;
+export const disableLegacyContext = false;
+export const disableSchedulerTimeoutBasedOnReactExpirationTime = false;
+export const enableTrustedTypesIntegration = false;
+export const disableTextareaChildren = false;
+export const disableModulePatternComponents = false;
+export const warnUnstableRenderSubtreeIntoContainer = false;
+export const warnAboutSpreadingKeyToJSX = false;
+export const enableComponentStackLocations = false;
```

Comment:
This file is copied from `ReactFeatureFlags.test-renderer.js` except for this flag.